File: client/skillz/src/app/models/consultants/person.ts
import { BaseAuditableEntity } from '../base/baseAuditableEntity';

export class Person extends BaseAuditableEntity {
  firstname: string;
  lastname: string;
  constructor(firstName, lastName) {
    super();
    this.firstname = firstName;
    this.lastname = lastName;
  }
}

File: server/Skillz/Skillz.Models/Entities/People/Person.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Skillz.Models.Entities.People
{
    public abstract class Person : BaseAuditableEntity
    {
        public string FirstName { get; set; }
        public string LastName { get; set; }

        public override string ToString()
        {

            return $"{FirstName} {LastName}";

        }
    }
}